pracownia_projektowa/prezentacja_wstepna/prezentacja_wstepna.tex

```diff
@@ -0,0 +1,286 @@
+%	PACKAGES AND THEMES
+%----------------------------------------------------------------------------------------
+
+\documentclass{beamer}
+
+\usepackage[T1]{fontenc}
+\usepackage[polish]{babel}
+\usepackage[utf8]{inputenc}
+\usepackage{lmodern}
+\usepackage{amsmath}
+
+\selectlanguage{polish}
+
+\mode<presentation> {
+	\usetheme{Copenhagen}
+}
+
+\usepackage{graphicx} % Allows including images
+\usepackage{booktabs} % Allows the use of \toprule, \midrule and \bottomrule in tables
+
+
+% Custom macros
+
+\newcommand{\red}[1]{
+	{ \color{red}{#1} }
+}
+
+\newcommand{\score}[2]{
+	\stackrel
+	{\red{#1}}
+	{#2}
+}
+
+\definecolor{links}{HTML}{2A1B81}
+\hypersetup{colorlinks,linkcolor=,urlcolor=links}
+
+
+%----------------------------------------------------------------------------------------
+%	TITLE PAGE
+%----------------------------------------------------------------------------------------
+
+\title
+[System obliczący wyniki wyborów]
+{System obliczający wyniki wyborów dla uogólnienia systemu k-Borda}
+
+\author
+[T. Kasprzyk, D. Ogiela, J. Stępak]
+{Tomasz Kasprzyk, \ Daniel Ogiela, \ Jakub Stępak}
+
+\institute
+[AGH]
+{
+Akademia Górniczo-Hutnicza
+
+Wydział Informatyki, Elektroniki i Telekomunikacji
+
+Katedra Informatyki 
+\newline \newline
+Projekt realizowany pod opieką \\dr. hab. inż. Piotra Faliszewskiego
+
+}
+\date{4 kwietnia 2016}
+
+%----------------------------------------------------------------------------------------
+
+\begin{document}
+
+\frame{\titlepage}
+
+\begin{frame}
+\frametitle{Plan}
+% TODO: zastanowić się czy to w ogóle potrzebne, IMHO nie i należy wywalić
+\tableofcontents
+\end{frame}
+
+%----------------------------------------------------------------------------------------
+%	PRESENTATION SLIDES
+%----------------------------------------------------------------------------------------
+
+%------------------------------------------------
+\section{Opis problemu}
+%------------------------------------------------
+
+\subsection{Wybory}
+
+\begin{frame}
+\frametitle{Wybory}
+Naszym zadaniem jest obliczenie wyników wyborów.
+Przez wybory możemy rozumieć zarówno „tradycyjne” wybory np. parlamentarne,
+ale także, na przykład, problem wyboru odpowiednich tytułów filmowych do
+systemów rozrywkowych samolotu, aby jak najwięcej pasażerów mogło wybrać
+coś dla siebie.
+\end{frame}
+
+%------------------------------------------------
+
+
+\begin{frame}
+\frametitle{Wybory formalnie}
+Wybory to para $E = (C, V)$,
+gdzie $C = {c_1, c_2, \ldots, c_m}$ to zbiór kandydatów,
+a $V = (v_1, v_2, \ldots, v_m)$ to ciąg wyborców.
+Każdy wyborca jest opisany przez swoje $preferencje$,
+które są ciągiem kandydatów w porządku od najbardziej pożądanego
+przez danego wyborcę do najmniej pożądanego. Rozpatrujemy wybory
+komitetów, więc otrzymujemy też liczbę $k$ będącą wielkością wybieranego komitetu.
+\end{frame}
+
+%------------------------------------------------
+
+\begin{frame}
+\frametitle{Przykładowe wybory}
```

Comment:
Zmiana tytułu na np. przykładowy przydział preferencji, jeśli Przykładowe wybory to zbiory S i V też pasowałoby określić